src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -656,7 +656,7 @@ private IOperation CreateBoundConversionOperation(BoundConversion boundConversio
             {
                 Lazy<IOperation> operand = new Lazy<IOperation>(() => Create(boundConversion.Operand));
                 SyntaxNode syntax = boundConversion.Syntax;
-                Conversion conversion = _semanticModel.GetConversion(syntax);
+                Conversion conversion = _semanticModel.GetTypeInfoForNode(boundConversion.Operand, boundConversion, null).ImplicitConversion;
```

Comment:
I started with VB, which doesn't have a similar property, so I didn't even look for such a thing. I'll switch CSharp to use that version.